mlflow/store/local_artifact_repo.py

```diff
@@ -1,13 +1,18 @@
 import distutils.dir_util as dir_util
 import shutil
 
+from six.moves import urllib
+
 from mlflow.store.artifact_repo import ArtifactRepository
-from mlflow.utils.file_utils import mkdir, list_all, get_file_info
+from mlflow.utils.file_utils import mkdir, list_all, get_file_info, parse_path
 from mlflow.utils.validation import path_not_unique, bad_path_message
 
 
 class LocalArtifactRepository(ArtifactRepository):
     """Stores artifacts as files in a local directory."""
+    def __init__(self, *args, **kwargs):
+        super(LocalArtifactRepository, self).__init__(*args, **kwargs)
+        self.artifact_dir = parse_path(self.artifact_uri)
```

Comment:
Should we check that the artifact_uri has expected prefix? Maybe we should standardize it so that it always has file:/// prefix?